tempo_embeddings/embeddings/vector_database.py

```diff
@@ -259,45 +263,55 @@ def get_records(
             where_doc = {"$contains": filter_words[0]}
         else:
             where_doc = {"$and": [{"$contains": w} for w in filter_words]}
-
-        result = collection.get(
-            where=where_obj, where_document=where_doc, include=include
+        # Query the collection
+        limit = limit if limit > 0 else None
```

Comment:
Minor suggestion to make the logic more comprehensible.

```suggestion
        if limit == 0:
            limit = None
```

While looking at this, however, I wonder if you need this conversion after all.